docs/.vitepress/config/ru.mts

```diff
@@ -163,6 +163,10 @@ function sidebarGuide(): DefaultTheme.SidebarItem[] {
                     text: 'Breaking Changes checks',
                     link: 'cli/breaking-changes/breaking-changes',
                 },
+                {
+                    text: 'Auto completion',
+                    link: 'cli/auto-completion/auto-completion',
+                },
```

Comment:
Mb in another block? For example introduction?

Reply:
ok, i'll change it